BitwardenShared/Core/Auth/Services/AuthServiceTests.swift

```diff
@@ -43,6 +45,7 @@ class AuthServiceTests: BitwardenTestCase { // swiftlint:disable:this type_body_
             appIdService: AppIdService(appSettingStore: appSettingsStore),
             authAPIService: authAPIService,
             clientService: clientService,
+            configService: configService,
```

Comment:
🎨 It feels strange to me to introduce the config service to the auth service, but have no tests around calling it.

Reply:
I still have several tests to add to the PR, that's why is not getting called.